# Where are the recent selective sweeps?
### How do we make sense of the surprisingly few loci under selection in the past 5,000 years?
**By Sasha Gusev — The Infinitesimal (Substack), Sep 14, 2024**
Source: https://theinfinitesimal.substack.com/p/where-are-the-recent-selective-sweeps

---

You have probably heard about the Tibetans who adapted to high altitude environments through mutation in a hypoxia pathway gene, or the Bajau "sea nomads" with genetic adaptations increasing spleen size to enable longer diving. It is tempting to imagine that such adaptive evolution is happening all around us in response to broad environmental differences: Germans evolving a punctuality gene, Russians developing an adaptation to tolerate high quantities of alcohol, Ukrainians developing, well, also an adaptation to tolerate alcohol, and so on. In reality, the reason these adaptations get so much attention is that they are exceptionally rare and highly specific to niche environments. To get a sense of what I'm talking about, the figure below shows representative results from a scan for genetic variants influencing a common trait (educational attainment) and a scan for genetic variants under recent selection (for any trait).

Each point is a test along the genome and the y-axis shows the statistical significance, with points above the dashed line being significant at the "genome-wide" level. On the left, the scan for genetic variants associated with educational attainment identified 3,952 significant loci. On the right, the scan for genetic variants under selection in the past 2,000-3,000 years identified a grand total of … three loci. These are not apples-to-apples comparisons since the data and methods have different dynamics, but they illustrate the extreme polygenicity we see for complex traits on the one hand, and the paucity of evidence for locus-specific selection on the other.

So many genetic levers that could influence a trait and yet so few of them get pulled by evolution. Why? To answer that question, let's face the future and walk backwards through history.

## What is natural selection?

Natural selection is the process by which differences in fitness across individuals lead to systematic changes in the frequency of fitness-associated alleles. People with a certain allele systematically have more children (fecundity selection) or have healthier babies (viability selection) and the allele frequency goes up. Offspring with a certain novel deleterious allele are systematically born non-viable, and so the frequency of that allele stays low and also pulls down the frequency of correlated variants with it (background selection).

Broadly speaking, there are two genomic features that are indicative of selection acting on a single locus:

**Allelic.** Where an unusual change in the allele frequency of a variant is observed using data from populations that have diverged. For example, a functionally important allele under positive/directional selection that is swept up to higher frequency in a certain population will exhibit a significant frequency difference with other populations (or with the same population measured at a different time, as with ancient DNA).

**Haplotypic.** Where unusual patterns of genetic diversity for an entire region are observed, typically by comparing to other regions (or some evolutionary model). For example, a functionally important gene that is constantly purging newly arising deleterious alleles (aka background selection) will exhibit unusually low levels of nearby neutral alleles relative to the rest of the genome, even if we never see the deleterious allele itself.

Thus, selection can be thought of as either the presence of an unexpected pattern across populations or the absence of an expected pattern across the genome. These two processes will often operate together, for example a positive selective sweep will increase the frequency of a new allele as well as all of the correlated variants, reducing the local haplotypic diversity.

Importantly, detecting these indicators of selection does not require one to know which trait selection is acting on (nor anything about the mechanism of selection), since they only pertain to unusual genetic patterns. This means that selection can be detected for any trait as long as the evolutionary process has been active in the population being analyzed.

## Selection in the past 200 years

In admixed populations, haplotypes from the ancestral populations are inherited in long chunks that are then broken down by recombination over generations. If one of the ancestral populations had a divergent allele that was under selection after admixture, we should expect to see an unusual excess of corresponding local ancestry in that region (or a depletion for selectively disadvantageous alleles).

One of the largest studies of directional admixture selection to date was conducted by Bhatia et al. (2014), and the title of the paper pretty much tells the whole story: "Genome-wide scan of 29,141 African Americans finds no evidence of directional selection since admixture". The authors ran an admixture scan looking for ancestry deviations and the results were thoroughly in the null. They evaluated six loci implicated in a previous, much smaller, admixture scan and showed that those are all null as well. Using simulations, they confirmed that their study has 95% power to identify loci under strong selection.

Finally, they investigated a handful of significant allelic differences previously observed between African Americans and other African populations: at HBB and CD36, both well-established targets of malaria; in the well-established major histocompatibility complex (MHC) locus, which is critical for immune function and autoimmune disease; and in the prostate cancer gene PSCA (which had been recently discovered by the same team). It is possible that these loci were evidence of recent selection that occurred after the forced migration from Africa but prior to admixture, or were somehow missed by the admixture scan. However, the authors also contest this hypothesis, showing that allele frequency differences at these loci can be much greater for pairs of populations within Africa. Such large frequency differences are most consistent with ancient selection occurring across Africa rather than post-migration.

If you go and read the paper you may notice a somewhat unusual tone. Most of the Results section, for example, is dedicated to explaining the technical pitfalls of prior admixture/selection scans and showing that previous findings were false positives or misinterpreted. The context is that this work was the culmination of nearly a decade of (sometimes acrimonious) debates over whether the transatlantic slave trade and other brutal forced migrations had fundamentally reshaped the genetic variation in admixed people. This study — large, well powered, and rigorously carried out by an experienced team — was thus intended to be a definitive and conclusive "no".

## Selection in the past 2,000 years

Detecting very recent selection in non-admixed populations is challenging because there has been insufficient time for a selected allele to distinguish itself from random variation (aka drift). However, recent positive selection at a locus is genetically equivalent to a reduction in the local population size and an increase in the haplotypic sharing across individuals. Field et al. (2016) noted that individuals who carry an allele under recent selection are, in essence, more genetically related in that region and will thus have fewer nearby "singletons" (new variants that are present in just a single individual) — a reduction that could be leveraged to detect selection. For a given site, they derive a Singleton Density Score (SDS) as the ratio of singleton likelihood statistics for the two alleles (derived and ancestral). A significant SDS statistic is then indicative of selection on the tested site. The key insight of this approach is that taking a ratio normalizes out many parameters that are otherwise very difficult to estimate, including local mutation rate, admixture, and demography. By focusing on singletons, SDS is also specifically powered for recent selective pressure; in contrast to other methods that will still find selection even if it has recently stopped.

SDS was applied to whole-genome sequencing data from ~3,000 individuals in the UK, where it was estimated to have high sensitivity for strong selection (selection parameter s>0.005) in the past 2,000-3,000 years (~100 generations). This analysis revealed a total of three loci (shown in the Manhattan plot at the start of this post): the well-established lactase persistence gene (LCT), related to the ability to process milk; the well-established major histocompatibility complex (MHC) locus that is critical for immune function; and the (at the time mostly unknown) WDFY4 gene, which has recently been shown to modulate the response to viral infection in mice. A secondary scan also showed some nominal evidence of enrichment at known hair and skin pigment loci. In sum, less than a handful loci under very recent strong selection, all linked to diet, autoimmunity, infection, or pigment.

## Selection in the past 5,000 years

The above selection scans leveraged patterns in data from contemporary populations, contrasting populations/haplotypes (admixture) or alleles/singletons (SDS). But the emergence of ancient DNA enabled novel approaches that could contrast allele frequencies across time. Mathieson et al. (2015) conducted a selection scan in 230 Eurasian ancient genomes from 300BC-6000BC, including Bronze Age and Neolithic farmers and various hunter-gatherer groups. To identify loci under selection, each variant was then simply tested for significant differences between ancient and modern populations given the modern/ancient admixture. Based on simulations, this data was sufficiently powered to identify strong selection within the past 100-200 generations (2500-5000 years).

Comparing allele frequencies between modern individuals and these ancient sub-populations revealed 12 loci under selection genome-wide. This again included the lactase (LCT) and MHC loci observed previously (indicative of sustained selection), as well as genes related to diet (FADS1), Vitamin D (DHCR7, NADSYN1), skin/eye pigment (SLC45A2, GRM5, HERC2), hair (EDAR), autoimmune conditions (SLC22A4, ATXN2), and infection (TLR). Within the past 5,000 years we continue to see the pattern of very few instances of strong selection, and all acting on diet, immunity/infection, and skin/hair.

It is quite surprising how arbitrary some of these associations are. The EDAR variant is one such example. It has been mechanistically associated with hair thickness, sweat glands, and facial structure; but not in ways that seem particularly selectively advantageous, more like subtle aesthetic differences. It also followed an unusual frequency trajectory. While it is nearly fixed in modern day East Asian and Native American populations, it appears to have arisen independently [Posth et al. (2018)]. Yet at the same time it is completely absent in ancient Japanese hunter-gatherer (Jomon) genomes from the past 3,000-5,000 years [Wang et al. (2021)]. So the variant was sufficiently advantageous to be swept up to fixation twice in different locations, while also being sufficiently dispensable to be absent in a geographically similar population. The HERC2 locus is another such example, in recessive form it leads to blue eyes, and it has swept up to near fixation in ancient hunter-gatherers and remained common in modern northern Europeans. In contemporary disease scans, it is weakly associated with some eye conditions and possibly skin cancer, but does not seem to have particularly useful or deleterious function beyond appearance. Did blue eyes really confer such a stark fitness advantage specific to northern Europe? Even the selection on LCT has its mysteries, with more recent data showing that selection started in Britain about a century prior to the rest of European samples [Patterson et al. (2022)], for reasons that are essentially unknown. Similarly unexpected are the genes that did not come up: no strong selection on APOE, a key Alzheimer's and longevity gene; no strong selection on FTO, one of the largest common effects on obesity and diabetes; or on genes recently found to substantially reduce the age of menopause and fertility. Within the context of autoimmune disease, IL23R is one of the largest known protective effects, and yet selection on this locus is either weak or outright neutral (depending on how you model admixture). In short, recent evolution seems to be picking winners and losers fairly arbitrarily.

The studies thus far have focused on finding individual loci under selection, but recent work from [Simon et al. (2024)] took the creative approach of quantifying the influence of selection on the entire spectrum of recent allele frequency differences. For data collected at multiple time points, they relate the variance in frequency changes to a combination of neutral drift (which shifts alleles randomly), selection (which causes frequencies to covary across time), and admixture (which mixes in frequencies from other source populations). This decomposition is averaged over all variants and time points to get a single parameter estimate across the entire genome. Applying the model to a large dataset of ancient and modern European genomes spanning back 5600 years, the authors find a surprisingly negligible contribution of selection relative to neutral drift and admixture. In multiple cases where frequency covariances were initially observed, they were largely explained by admixture. Even restricting to regions of the genome where selection is expected to be more prominent — those that are functionally important and/or have low recombination rates — the contribution of selection to frequency change was still a paltry ~3%. The authors conclude that "gene flow is the dominant force changing allele frequencies in the recent history of European human populations".

These findings do not contradict the earlier results from Mathieson et al., but suggest that those 12 loci are more likely to be outliers in the selective landscape rather than the tip of some genome-wide iceberg.

## So … where are the selective sweeps?

A lot has happened to humans in the past 5,000 years and yet, from an evolutionary perspective, it looks as though very little has happened. Based on ancient and modern data we see two broad and somewhat contradictory patterns. On the one hand, very intense but short term stress (~200 years) is not sufficient to induce strong selection. The transatlantic slave trade was one of the most brutal and severe multi-generational abuses of humans in recent history, and yet it left no apparent evolutionary trace in modern day populations. On the other hand, the few loci that have come up in scans for recent selection are largely acting on simple processes related to diet, appearance, or infection and (beyond LCT and the MHC) the underlying genes seem fairly arbitrary. Based on genome-wide estimates, the contribution of recent selection is negligible while admixture and migration appears to be a substantial driver of allele frequency change. What models of recent evolution could explain these patterns?

It is possible that neutrality — the null hypothesis of evolution — is simply the rule, and instances of locus-specific selection are chance exceptions. Individual loci either do not substantially contribute to fitness in humans in the recent period, or the etiology of fitness changes sufficiently quickly such that sustained sweeps cannot get traction, or a complex multi-dimensional fitness landscape neutralizes the effects of individual loci. The fact that most common traits are extremely polygenic could either imply that there are many opportunities for strong individual adaptations to arise or that any one allele has such a negligible fitness effect that it is effectively neutral. Multiple lines of evidence now point to the latter, as has been hypothesized for some time.

For the few loci that are observed to be under strong selection, they may be driven by indirect environmental forces with genes along for the ride. Say the HERC2 blue eyed allele arises in an isolated tribe; for cultural reasons, members of the tribe mate only with other blue eyes and also, due to pure dumb luck, successfully pillage and conquer their neighbors over the course of many centuries — driving HERC2 variants to near fixation during that time. In this example, blue eyes were not the drivers of evolutionary success, but they came along for the ride through a combination of rapidly changing population dynamics, phenotypic assortative mating, and good luck. Purely decorative spandrels. This phenomena could explain why seemingly mild effects related to appearance and diet changed in frequency so rapidly; after all, these are the factors that link people culturally and culture can shape genes.

Ancient DNA has revealed that population history over the past several thousand years was surprisingly dynamic, with populations mixing and displacing each other frequently and sometimes rapidly. In the recent historical period, there is clear evidence of both long-distance migration and sustained population structure, as would be consistent with highly non-random mating [Antonio et al. (2024)]. These may be the components of a dynamic yet largely neutral process, where a handful of variants get swept up mostly due to dumb luck.

---

## Footnotes

1. "Figure 1 displays the average local ancestry at each SNP and indicates no genome-wide-significant deviation in local ancestry." ~ Bhatia et al.

2. "When we used a threshold of 3 SDs in our data, six loci showed significant deviations. None of these overlap those reported by Jin et al. (see Table S5), suggesting that reported signals of selection after admixture are likely to be false positives because of an insufficient correction for multiple tests." ~ Bhatia et al.

3. "Our results suggest that selection stronger than s_anc > 0.019 since admixture can be ruled out" ~ Bhatia et al.

4. "Thus, we believe that selection in Africa rather than post-Africa is the most likely explanation for most of the observed frequency differences between African Americans and YRI … Overall, we conclude that there is no locus with genome-wide-significant evidence of selection influencing ancestry in African Americans after their ancestors left Africa and that genome-wide-significant evidence of population differentiation is likely to be best explained by selection in Africa." ~ Bhatia et al.

5. For example, see the exchange of letters to the editor between Price and Tang regarding an earlier admixture study of Puerto Ricans.

6. This may seem like a simple statistic, but it was a very elegant solution to a problem that many other methods had struggled with. Oftentimes the key insight is not a complicated algorithm, but understanding how to marginalize out the need for a complicated algorithm.

7. Field et al. also conducted a broader scan for polygenic selection on complex traits, which revealed many associations and generated a great deal of excitement. However, this secondary test did not have the benefit of being immune to population stratification, and was subsequently shown to be largely or even entirely confounded by many of the same authors [Berg et al.; Sohail et al.]. These artifactual findings merit a separate and longer discussion.

8. Neutral drift is expected to inflate the variance across all variants tested and so was accounted for post hoc with a genomic correction. This is not an ideal solution if drift differs across the genome due to, for example, background selection and might inflate some of the results. Here again we see why the "internal control" of SDS is so appealing.

9. A limitation with this approach is that, in contrast to the SDS test, it does need to model the changing population dynamics and therefore makes some strong assumptions on the estimation of admixture and the availability of reference populations.

10. "In sum, we see little evidence, in either transect, of linked selection in the covariances in allele frequency change between time intervals, suggesting that having accounted for admixture, much of the residual change across time intervals is due to drift-like sampling processes" ~ Simon et al.

11. "As I discussed at length, we do now have compelling examples of the various forms of positive selection acting in humans: including hard and soft sweeps, and ancient balancing selection. However, my personal reading of the data is that strong hard selection on individual loci has been rare in the human genome during the past ∼200,000 years when we can best detect it. Many of the exceptions where we do see sweep signals are at genes where a single protein plays an exceptional role in some process–for example Duffy, which serves as a specific receptor for vivax malaria; or lactase which plays an essential role in digesting lactose." ~ Jonathan Pritchard in his recent book; see also Pritchard et al. (2010).

12. I'm stealing this expression from another excellent post by David Hugh-Jones about different models of heritability, where genes come along for the ride in a different but related way.
