If the direction of selection keeps flipping (thus making it harder to identify sweeps) then how does this method do any differently or better in  terms of distinguishing drift?

Your work  has highlthed how big a deal gene flow and population changes/replacement have been through history. Does that count as selection? What if some gene is relevant to why one population wins out over another?

What are the consequences of humanity’s effective population size only being 10,000?

“You read that right, genetic variants influencing behavioral traits do not appear to be enriched for loci that are under selection.” \- is this true?

Why do so few of the previous findings replicate:

Mathieson et al. (most conservative prior study): 10/11 replicate. Good.

Le et al.: 9/22 replicate. Mixed.

Kerner et al.: 14/123 replicate. Bad.

Field et al. sub-threshold loci: 1/34 replicate. Terrible.

How do you deal with background selection?

### Outline of blog post recap

Say that you want to figure out how much selection has been going in the last 10,000 years of human history. How would you figure that out?

Here’s one obvious idea: find all the 10,000 year old bones, find allele frequencies, and compare them to allele frequencies today

Problem: 1\. Doesn’t account for population admixture

Solution: expect it to within the range predicted by population admixture rates \- this is what Mathieson et al. did with 230 ancient genomes

Problem: there’s an initial problem of sample size (you only have 300 sample) \- but even if you solve that, you still have to deal with drift and background selection:

Allele frequency can change due to one of 3 different things:

Selection

Drift 

Drift is really bad because it scales with generations / effective population size \- and the plausible drift scales inversely to the effective population size \- which is really small for WHG

Furthermore, Background selection can make it look liek certain locations are under selection which aren’t

You can try other clever things \- for example, since genetic regions under selection will be spreading more rapidly through the population, you can check how recent the most recent common ancestor for different genetic regions is (the more recent, the more the area was under selection). The problem with this is that  Or you can see how many ultra rare mutations are near a selected allele \- the fewer there are, the more recently the allele arose \- and if a recently arised allele has achieved high freuenzy, then it was probably under selection \- the problem with this is that it can only tell recent selection \- if selection fluctuated or reversed, you wouldn’t detect it. Gusev discusses a bunch more in this blog post. 

## What these previous results kept finding

* Recent selection is quite limited \- only TK loci  
* It’s mainly forced 

gene drift makes the null hypothesis really wide 

